src/components/Layout.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div>
+    <div class="col-12 col-lg-3 float-left hide-sm hide-md hide-lg">
```

Comment:
This should not appear if the slot doesn't exist on Vue2 you can do like this
```suggestion
    <div v-if="$slots['sidebar-left']" class="col-12 col-lg-3 float-left hide-sm hide-md hide-lg">
```